### PR TITLE
Add flag for serverless unpacking of non-string values in a playbook

### PR DESCRIPTION
**Pull request by xiaohtwilio**
…olean variable

Add apb_parseIntBool wrapper to allow playbook YAML render int and boolean variable

Maintainer edits:
- changed `apb_parseIntBool` to `render_nonstring_value` (open to discussion about finding a final name)
- added tests
- added a constants file
- updated readme

**Contributing to Twilio**

> All third-party contributors acknowledge that any contributions they provide will be made under the same open-source license that the open-source project is provided under.

- [x] I acknowledge that all my contributions will be made under the project's license.

### CODE REVIEW

File: lib/apb.js [hunk 1]
```diff
@@ -6,6 +6,9 @@ const DEFAULT_RETRY = {
   "BackoffRate": 2
 }
 
+const PARSE_SELF_NAME = "render_nonstring_value"
```
Comment (noxasaxon) [inline lib/apb.js:9]: name of the flag/function

File: test/test.js [hunk 2]
```diff
@@ -241,4 +241,28 @@ describe('apb', () => {
 
   })
 
+  describe('#render_nonstring_value', () => {
```
Comment (noxasaxon) [inline test/test.js:244]: tests for the new flag

File: test/mocks/playbooks/general.js [hunk 3]
```diff
@@ -148,5 +148,35 @@ module.exports = {
                 "all": false
             }
         }
+    },
+    pb_parse_nonstring: {
```
Comment (noxasaxon) [inline test/mocks/playbooks/general.js:152]: new test playbook

File: README.md [hunk 4]
```diff
@@ -52,6 +52,16 @@ To automatically add a Task Failure Handler to each `Task` state that will trigg
 }
 ```
 
+To use serverless unpacking for nonstring values, use the `render_nonstring_value()` flag in your `playbook.json`. The example below is referencing an Int type in the serverless.yml custom variables definition:
```
Comment (ubaniabalogun) [inline README.md:55]: can we perfiix this to `apb_render_nonstring_value`? Primary concern here is if we accidentally add a `render_nonstring_value` function later down the line in SOCless via Jinja templates, we'd end up with conflicting behaviour. Unlikely but could happen

**Review by ubaniabalogun — CHANGES_REQUESTED** [review]

**Review by ubaniabalogun — APPROVED** [review]

**Pull request merged**